### PR TITLE
Add project chiquito

### PR DESCRIPTION
**Pull request by leolara**

### CODE REVIEW

File: data/projects/chiquito.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { ProjectInterface } from "@/lib/types"
+
+const description = `
+Chiquito is a high-level structured language for implementing zero knowledge proof circuits, currently being implemented in the DSL Working Group of PSE. It is a state machine zk-language that provides better syntax and abstraction. It can automatically compiler arbitrary boolean expressions for state machine transitions. It can optimise the resulting arithmetization. Chiquito has a Halo2 backend, which is a low level zkDSL that writes circuits using the PLONKish arithmetization and is working on supporting additional backends. Chiquito circuits can be written using both Python and Rust. 
+`
+
+export const chiquito: ProjectInterface = {
```
Comment (kalidiagne) [inline data/projects/chiquito.ts:7]: @leolara this needs to be added inside the `projects.ts` file to see the project in the list

**Review by kichong — APPROVED** [review]

**Review by AtHeartEngineer — APPROVED** [review]

File: data/projects/chiquito.ts (hunk 1 above)
Comment (leolara) [reply data/projects/chiquito.ts:7]: Done

**Comment by leolara** [comment]
I have added chiquito to projects.ts and solved conflict with main.

Vercel does not let me merge though.

File: data/projects.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ import { zkitter } from "./projects/zkitter"
 import { zkml } from "./projects/zkml"
 import { Zkopru } from "./projects/zkopru"
 import { zkp2p } from "./projects/zkp2p"
+import { chiquito } from "./projects/chiquito"
```
Comment (kalidiagne) [inline data/projects.ts:42]: @leolara only one missing thing: this needs to be added to the `projects` (line 55-56) list below. 

here a detailed overview of the steps to follow https://github.com/privacy-scaling-explorations/website-v2/blob/main/data/projects/README.md

**Comment by leolara** [comment]
@kalidiagne please check again

**Comment by kalidiagne** [comment]
> @kalidiagne please check again

Perfect! merging now!

**Review by kalidiagne — APPROVED** [review]

**Pull request merged**